### DISCUSSION TITLE
After updating the latest nuxt 3.8 version, composables auto import is not working.

### DISCUSSION THREAD

**Discussion by SaidGandur01** [nuxt/nuxt#24056] [Questions]
For some reason after updating the nuxt 3.8 version, my project is not auto importing the composables unless i go to each file and add the import itself.

By the way the current nuxt version is "nuxt": "^3.5.3", so I'm not sure why this is happening,

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏
